site/content/docs/5.3/examples/blog/index.html

```diff
@@ -56,11 +69,14 @@ <h1 class="display-4 fst-italic">Title of a longer featured blog post</h1>
     <div class="col-md-6">
       <div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative">
         <div class="col p-4 d-flex flex-column position-static">
-          <strong class="d-inline-block mb-2 text-primary">World</strong>
+          <strong class="d-inline-block mb-2 text-primary-emphasis">World</strong>
           <h3 class="mb-0">Featured post</h3>
           <div class="mb-1 text-body-secondary">Nov 12</div>
           <p class="card-text mb-auto">This is a wider card with supporting text below as a natural lead-in to additional content.</p>
-          <a href="#" class="stretched-link">Continue reading</a>
+          <a href="#" class="icon-link gap-1 icon-link-hover stretched-link">
+            Continue reading
+            <svg class="bi"><use xlink:href="#chevron-right"/></svg>
```

Comment:
There's an issue with the right chevron which seems a bit large 
![Screenshot 2023-02-16 at 17 23 27](https://user-images.githubusercontent.com/17381666/219426455-ae75a620-bf62-4eaa-a1d6-2fbe2215495b.png)

Reply:
Hmm where you seeing this? Looks good for me.

Reply:
Don't remember, might have been a cache issue :) Resolving this comment.